titan/features/prep.py

```diff
@@ -139,7 +139,7 @@ def get_acquisition_risk_multiplier(self, time: int, interaction_type: str):
                     return 1.0 - params.prep.efficacy.adherent
                 else:
                     return 1.0 - params.prep.efficacy.non_adherent
-            elif self.type == "Inj" and not self.adherent:
+            elif self.type == "Inj" and self.adherent:
```

Comment:
what does it mean to be non-adherent with injectable prep?

Reply:
I have no idea! Legacy code, but it shouldn't depend on adherence